src/Chainweb/BlockHeaderDB/RestAPI.hs

```diff
@@ -357,8 +356,6 @@ someBlockHeaderDbApis v = mconcat . fmap (someBlockHeaderDbApi v)
 -- -------------------------------------------------------------------------- --
 -- BlockHeader Event Stream
 
-newtype HeaderStream = HeaderStream Bool
```

Comment:
Why was this deleted?

Reply:
I mentioned it in one of my own comments.